### ISSUE TITLE
Disappearing armor When using shader

### ISSUE THREAD

**Issue by HatelessHeart** [sixthsurge/photon#178]
### What happened?

 I've tried just about everything. 
![Screenshot 2024-06-11 025933](https://github.com/sixthsurge/photon/assets/149926055/34d39a9f-03ff-4c1d-96b7-cc7bf04902c5)
![Screenshot 2024-06-11 025918](https://github.com/sixthsurge/photon/assets/149926055/1dbf0dff-c20b-47ec-b988-8332c4b9acbb)

### Minecraft Version

1.20.1

### Which shader mod are you using?

Iris

### Shader mod version

photon-main.zip

### Shader settings

```conf
#Tue Jun 11 02:48:01 CDT 2024
STARS_COVERAGE=0.80
shadowDistance=96.0
VOXEL_VOLUME_SIZE=64
COLORED_LIGHTS=true
AUTO_EXPOSURE=AUTO_EXPOSURE_SIMPLE
WAVING_LEAVES=false
VIGNETTE=false
HANDHELD_LIGHTING=true
TAAU=true
WAVING_PLANTS=false
NORMAL_MAPPING=true
BLOOM_INTENSITY=1.15
AIR_FOG_CLOUDY_NOISE=false
CAS_INTENSITY=0.72
CAVE_FOG=false
shadowMapResolution=1536
AURORA_BRIGHTNESS=1.18
```

**Comment by Plain-Second** [comment]
me too

**Comment by Plain-Second** [comment]
This problem still seem to have not been fixed, do you still have this problem?

**Comment by HatelessHeart** [comment]
> This problem still seem to have not been fixed, do you still have this problem?

Have they updated the shader?

**Comment by Plain-Second** [comment]
> > 这个问题似乎还没有解决，你还有这个问题吗？
> 
> 他们更新了着色器吗？
yes, I downloaded the latest file. I've found the cause in a previous issue(#158 ) It's because of a conflict with mod(Shoulder Surfing Reloaded).  Changing to other mods will solve this problem. But I'd like you to fix this conflict, because I didn't  meet this conflict when I was using complementary r5.

**Comment by Plain-Second** [comment]
> > 这个问题似乎还没有解决，你还有这个问题吗？
> 
> 他们更新了着色器吗？

thanks very very much
